package.yaml

```diff
@@ -47,26 +44,6 @@ dependencies:
   - vector >= 0.12.1.2
 library:
   source-dirs: src
-  other-modules: Bitcoin.Keys.Extended.Internal
```

Comment:
This is maybe a questionable thing, but tbh, I really dislike `other-modules` as a thing. Like, what if I want the fingerprint's `Word32` or whatever. Just lemme do it.

Reply:
I do not like representing `Fingerprint` as a `Word32`.  How about we just change it to `SizedByteArray 4 ByteString`?  Then I would be more comfortable exposing `Fingerprint` internals, and maybe getting rid of `Bitcoin.Keys.Extended.Internal`.

Reply:
IMHO, that sounds out of scope for this changeset, but certainly an interesting thought.

Reply:
Agreed different changeset. I think exposing Internal modules is common practice in our ecosystem. People know the footguns they entail, let's just expose it.